docs/contrib.md

```diff
@@ -0,0 +1,51 @@
+(contrib)=
```

Comment:
Remove this. It can be linked to with `{doc}`contrib` if needed elsewhere.

Reply:
@davidism I would prefer leaving this as is because other places in the docs it uses this link and removing it will break them.